pkg/command/command.go

```diff
@@ -78,16 +76,41 @@ func (c *Command) RunExecutable(executable string, params ...string) error {
 		cmd.Dir = c.dir
 	}
 
-	if len(c.env) > 0 {
-		cmd.Env = c.env
-	}
+	setEnvironment(cmd, c.env)
 
 	if err := runCmd(cmd, _out, _err); err != nil {
 		return errors.Wrapf(err, "running command '%v' failed", executable)
 	}
 	return nil
 }
 
+func setEnvironment(cmd *exec.Cmd, env []string) {
```

Comment:
I would suggest to name this `appendEnvironment`.

Reply:
done